packages/documentation/src/components/sidebar/SidebarItems.jsx

```diff
@@ -1,45 +1,12 @@
 import React from 'react';
-import { Link, StaticQuery, graphql } from 'gatsby';
+import { Link } from 'gatsby';
 
 /**
  * Sidebar
  *
  * @param {props} props
  */
 
-function ComponentList() {
```

Comment:
We no longer need to display the components in the sidebar on the `Visual Design` page

This was what we had previously:

![image](https://user-images.githubusercontent.com/2008881/105244189-045c2f80-5b25-11eb-8220-df3ede34d180.png)